apps/bfd-shared-utils/src/main/java/gov/cms/bfd/sharedutils/database/DatabaseApplicationUrl.java

```diff
@@ -0,0 +1,25 @@
+package gov.cms.bfd.sharedutils.database;
+
+/** DatabaseApplicationUrl sets the application name for a jdbc url. */
+public class DatabaseApplicationUrl {
+  /** Database application test name constant value. */
+  public static final String DATABASE_APPLICATION_NAME = "bfdserver";
+
+  /**
+   * Sets the db url to include the application name if its not a hsqldb.
+   *
+   * @param databaseUrl database url to use
+   * @return formatted string of url with application name
+   */
+  public static String includeApplicationNameInDbUrl(String databaseUrl) {
```

Comment:
DatabaseTestUtils is calling this anyhow; why not just have this method live there instead? We don't gain a lot by having this be its own class.